unique_toolkit/unique_toolkit/content/service.py

```diff
@@ -27,6 +30,8 @@ def search_content_chunks(
         search_string: str,
         search_type: ContentSearchType,
         limit: int,
+        reranker: Optional[RerankerConfig] = None,
```

Comment:
I suggest to add here the "_config" as suffix as it describes the variable in a better name.